src/mock-db-data/library-css/components/accordion.ts

```diff
@@ -1,138 +1,46 @@
-      </div>
-    </div>
-  </div>
-  <div class="accordion-item">
-    <h2 class="accordion-header" id="headingTwo">
-      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
-        Accordion Item #2
-      </button>
-    </h2>
-    <div id="collapseTwo" class="accordion-collapse collapse" aria-labelledby="headingTwo" data-bs-parent="#accordionExample">
-      <div class="accordion-body">
-        <p>This is the second item's accordion body.</p>
-      </div>
-    </div>
-  </div>
-  <div class="accordion-item">
-    <h2 class="accordion-header" id="headingThree">
-      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
-        Accordion Item #3
-      </button>
-    </h2>
-    <div id="collapseThree" class="accordion-collapse collapse" aria-labelledby="headingThree" data-bs-parent="#accordionExample">
-      <div class="accordion-body">
-        <p>This is the third item's accordion body.</p>
-      </div>
-    </div>
-  </div>
-</div>`,
-  documentationMarkdown: `
-<p>The accordion is using Bootstrap 5 markup. For functionality you can either include Bootstrap JS to your project or write your own. Remember to include the <abbr title="accessibility">a11y</abbr> principles.</p>
-
-<p>If you decide to write your own functionality, you might concider changing the <code>data-bs-</code> prefixes to <code>data-fhi-</code></p>
-
-<p>Google search: <a href="https://www.google.com/search?q=bootstrap+accordion+site%3Agetbootstrap.com">Bootstrap Accordion</a></p>`
-  ,
+export const Accordion: LibraryExample[] = [{
+  title: itemTitle,
+  type: LibraryItemType.ngBootstrap,
+  codeMarkdown: `
+<ngb-accordion [closeOthers]="true" #acc="ngbAccordion"
+               class="accordion-flush" activeIds="custom-id-0">
+  <ngb-panel id="custom-id-0" title="Item #1">
+    <ng-template ngbPanelContent>
+      <p>Item #1 content</p>
+    </ng-template>
+  </ngb-panel>
+  <ngb-panel id="custom-id-1" title="Item #2">
+    <ng-template ngbPanelContent>
+      <p>Item #2 content</p>
+    </ng-template>
+  </ngb-panel>
+  <ngb-panel id="custom-id-2" title="Item #3">
+    <ng-template ngbPanelContent>
+      <p>Item #3 content</p>
+    </ng-template>
+  </ngb-panel>
+</ngb-accordion>`,
+  documentationHtml: `
+<p>
+  Bootstrap documentation for
+  <a href="${BootstrapBaseUrl}/${itemTitle.toLocaleLowerCase()}">Accordion</a>
+</p>
+<p>
+  The accordion is implemented as a
+  <a href="${NgBootstrapBaseUrl}/${itemTitle.toLocaleLowerCase}">ngBootstrap Accordion</a>
```

Comment:
Same issue as above. I forgot to point that out.

Reply:
Hehe, my bad, had to run to meeting, so didn't look before jumping 😉